docs/benchmarking-workflow-corefx.md

```diff
@@ -0,0 +1,253 @@
+# Benchmarking workflow for CoreFX
+
+## Table of Contents
+
+- [Introduction](#Introduction)
+  - [Code Organization](#Code-Organization)
+  - [CoreFX Prerequisites](#CoreFX-Prerequisites)
+- [Preventing Regressions](#Preventing-Regressions)
+- [Solving Regressions](#Solving-Regressions)
+  - [Repro Case](#Repro-Case)
+  - [Profiling](#Profiling)
+  - [Running against Older Versions](#Running-against-Older-Versions)
+- [Local CoreCLR Build](#Local-CoreCLR-Build)
+- [Benchmarking new API](#Benchmarking-new-API)
+  - [Reference](#Reference)
+  - [PR](#PR)
+
+## Introduction
+
+This repository is **independent of CoreFX build system.** All you need to get the benchmarks running is to download dotnet cli or use the python script. Please see [Prerequisites](./prerequisites.md) for more.
+
+If you are not familiar with BenchmarkDotNet or this repository you should read the [Microbenchmarks Guide](../src/benchmarks/micro/README.md) first. It's really short and concise, we really encourage you to read it.
+
+### Code Organization
+
+All CoreFX benchmarks which have been ported from CoreFX repository belong to the corresponding folders: `corefx\$namespace`. The directory structure is following (some folders have been ommited for brevity):
+
+```log
+PS C:\Projects\performance\src\benchmarks\micro> tree
+├───corefx
+│   ├───System
+│   ├───System.Collections
+│   ├───System.ComponentModel.TypeConverter
+│   ├───System.Console
+│   ├───System.Diagnostics
+│   ├───System.Globalization
+│   ├───System.IO.Compression
+│   ├───System.IO.FileSystem
+│   ├───System.IO.MemoryMappedFiles
+│   ├───System.IO.Pipes
+│   ├───System.Linq
+│   ├───System.Memory
+│   ├───System.Net.Http
+│   ├───System.Net.Primitives
+│   ├───System.Net.Sockets
+│   ├───System.Numerics.Vectors
+│   ├───System.Runtime
+│   ├───System.Runtime.Extensions
+│   ├───System.Runtime.Numerics
+│   ├───System.Runtime.Serialization.Formatters
+│   ├───System.Security.Cryptography
+│   ├───System.Security.Cryptography.Primitives
+│   ├───System.Text.Encoding
+│   ├───System.Text.RegularExpressions
+│   ├───System.Threading
+│   ├───System.Threading.Channels
+│   ├───System.Threading.Tasks
+│   ├───System.Threading.Tasks.Extensions
+│   ├───System.Threading.ThreadPool
+│   ├───System.Threading.Timers
+│   └───System.Xml.XmlDocument
+```
+
+During the port from xunit-performance to BenchmarkDotNet the namespaces, type and methods names were not changed. The exception to this rule are all `System.Collections` ([#92](https://github.com/dotnet/performance/pull/92)) and `Span<T>` ([#94](https://github.com/dotnet/performance/pull/94)) benchmarks which got rewritten to utilize the full capabilities of BenchmarkDotNet.
```

Comment:
Add a comma: During the port from xunit-performance to BenchmarkDo**tNet, the** ...